packages/block-editor/src/components/colors-gradients/dropdown.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalItemGroup as ItemGroup,
+	__experimentalItem as Item,
+	__experimentalHStack as HStack,
+	FlexItem,
+	ColorIndicator,
+	Dropdown,
+} from '@wordpress/components';
+import { isRTL } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ColorGradientControl from './control';
+
+export default function ColorGradientSettingsDropdown( {
+	colors,
+	gradients,
+	disableCustomColors,
+	disableCustomGradients,
+	__experimentalHasMultipleOrigins,
+	__experimentalIsRenderedInSidebar,
+	enableAlpha,
+	settings,
+} ) {
+	let dropdownPosition;
+	if ( __experimentalIsRenderedInSidebar ) {
+		dropdownPosition = isRTL() ? 'bottom right' : 'bottom left';
+	}
+
+	return (
+		<ItemGroup
+			isBordered
+			isSeparated
+			className="block-editor-panel-color-gradient-settings__item-group"
+		>
+			{ settings.map( ( setting, index ) => (
+				<Dropdown
+					key={ index }
+					position={ dropdownPosition }
+					className="block-editor-panel-color-gradient-settings__dropdown"
+					contentClassName="block-editor-panel-color-gradient-settings__dropdown-content"
+					renderToggle={ ( { isOpen, onToggle } ) => {
+						return (
+							<Item
```

Comment:
I don't remember if this exact situation already came up in another PR (cc @jasmussen ), but have you tried to have the `ItemGroup` inside `Dropdown` instead? ie.

```jsx
<Dropdown>
  <ItemGroup>
    <Item>
```

Reply:
We did try that, yes, and there were ome problems with it. I can't recall them exactly, they are noted somewhere on one of Jorges PRs, and it's something to follow up on, but was acceptable for now. Thanks for looking, and thanks Staci!